sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -1411,7 +1411,8 @@ class SessionCatalog(
     parts.map { part =>
       if (part.storage.locationUri.isDefined && !part.storage.locationUri.get.isAbsolute) {
         val partPath = new Path(new Path(tbl.location), new Path(part.storage.locationUri.get))
-        val qualifiedPartPath = makeQualifiedPath(CatalogUtils.stringToURI(partPath.toString))
+        val partitionUri = CatalogUtils.stringToURI(partPath.toString, "partition location")
```

Comment:
What is the difference between `partition location` and `partition` used elsewhere?